lighthouse-core/audits/deprecations.js

```diff
@@ -59,6 +60,23 @@ class Deprecations extends Audit {
           }, log.entry);
         });
 
+    /**
+     * @return {!DetailsRenderer.CodeDetailsJSON} details
+     */
+    function makeDepractionsV2(entries) {
```

Comment:
Didn't think it was worth elevating to the class. This func is going away when we ditch extended info.